streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -86,36 +88,38 @@
      */
     public <K, V> KStream<K, V> stream(final AutoOffsetReset offsetReset,
                                        final String... topics) {
-        return stream(offsetReset, null, null, topics);
+        return stream(offsetReset, null, null, null, topics);
     }
 
+
     /**
      * Create a {@link KStream} from the specified topic pattern.
-     * The default {@code "auto.offset.reset"} strategy and default key and value deserializers as specified in the
-     * {@link StreamsConfig config} are used.
+     * The default {@code "auto.offset.reset"} strategy, default {@link TimestampExtractor}, and default key and value
+     * deserializers as specified in the {@link StreamsConfig config} are used.
      * <p>
      * If multiple topics are matched by the specified pattern, the created {@link KStream} will read data from all of
      * them and there is no ordering guarantee between records from different topics.
```

Comment:
I just realized, that we use different wording for topics as array of Strings and topic pattern:
"there is no ordering guarantee" vs "there are no ordering guarantees" -- I think we should clean this up for consistency. Would you mind to add this fix to this PR? The singular version sounds better, IMHO.